Firestore/core/test/firebase/firestore/local/remote_document_cache_test.mm

```diff
@@ -63,8 +63,8 @@
 /**
  * Extracts all the actual MaybeDocument instances from the given document map
  */
-std::vector<MaybeDocument> ExtractDocuments(
-    const OptionalMaybeDocumentMap& docs) {
+template <typename MapType>
+std::vector<MaybeDocument> ExtractDocuments(const MapType& docs) {
```

Comment:
This is a fix for https://github.com/firebase/firebase-ios-sdk/pull/4136/files/043cdc45b02b9d365aeadbe9b0ff9dffdb03aa55#diff-c4c9d440fb2b6a0d5966b70d73695c31 (a PR I merged into indexfree-master before it was merged to master)